gslib/commands/cp.py

```diff
@@ -631,147 +615,141 @@
+  -R, -r         The ``-R`` and ``-r`` options are synonymous. They enable directories,
                  buckets, and bucket subdirectories to be copied recursively.
-                 If you neglect to use this option for an upload, gsutil will
-                 copy any files it finds and skip any directories. Similarly,
-                 neglecting to specify this option for a download will cause
-                 gsutil to copy any objects at the current bucket directory
-                 level, and skip any subdirectories.
+                 If you don't use this option for an upload, gsutil copies objects
+                 it finds and skips directories. Similarly, if you don't
+                 specify this option for a download, gsutil copies
+                 objects at the current bucket directory level and skips subdirectories.
 
-  -s <class>     The storage class of the destination object(s). If not
+  -s <class>     Specifies the storage class of the destination object. If not
                  specified, the default storage class of the destination bucket
-                 is used. Not valid for copying to non-cloud destinations.
+                 is used. This option is not valid for copying to non-cloud destinations.
 
-  -U             Skip objects with unsupported object types instead of failing.
-                 Unsupported object types are Amazon S3 Objects in the GLACIER
+  -U             Skips objects with unsupported object types instead of failing.
+                 Unsupported object types include Amazon S3 objects in the GLACIER
                  storage class.
 
-  -v             Requests that the version-specific URL for each uploaded object
-                 be printed. Given this URL you can make future upload requests
-                 that are safe in the face of concurrent updates, because Google
-                 Cloud Storage will refuse to perform the update if the current
+  -v             Prints the version-specific URL for each uploaded object. You can
+                 use these URLs to safely make concurrent upload requests, because
+                 Cloud Storage refuses to perform an update if the current
                  object version doesn't match the version-specific URL. See
                  "gsutil help versions" for more details.
 
   -z <ext,...>   Applies gzip content-encoding to any file upload whose
-                 extension matches the -z extension list. This is useful when
-                 uploading files with compressible content (such as .js, .css,
-                 or .html files) because it saves network bandwidth and space
-                 in Google Cloud Storage, which in turn reduces storage costs.
+                 extension matches the ``-z`` extension list. This is useful when
+                 uploading files with compressible content such as .js, .css,
+                 or .html files, because it reduces network bandwidth and storage
+                 costs.
```

Comment:
"because it reduces network bandwidth and storage sizes. This can improve both performance and reduce costs."

Reply:
Done.